src/tests/_internal/core/models/test_resources.py

```diff
@@ -1,7 +1,6 @@
 import pytest
-from pydantic import ValidationError, parse_obj_as
-
 from dstack._internal.core.models.resources import GPU, ComputeCapability, Memory, Range
```

Comment:
Imports from dstack should come after third-party imports. Fix imports in tests.

Reply:
Fixed. The option [known-first-party](https://docs.astral.sh/ruff/settings/#isort-known-first-party) is configured for this.